_posts/en/newsletters/2019-01-08-newsletter.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Bitcoin Optech Newsletter #28'
+permalink: /en/newsletters/2019/01/08/
+name: 2019-01-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a new maintenance release of Bitcoin
+Core, describes continued discussion about new signature hashes, and
+links to a post about possible economic barriers to LN payments crossing
+different currencies.  Descriptions of notable code changes in popular
+Bitcoin infrastructure projects are also provided.
+
+## Action items
```

Comment:
Nit: I don't know if we're strict we are with the Markdown headers, but in the case we are: should we remove one `#` in the header level (from `##` to `#`) to make it so that the first header is always a top level header?

Reply:
When using Jekyll, the `title` field in the YAML sets the the content of the rendered page's `<title>` and `<h1>` tags, so our top-level section are semantically correct for using `##` to generate `<h2>`s.

Reply:
Makes perfect sense! Will not report this going forward :-)